code/drasil-printers/lib/Language/Drasil/TeX/Print.hs

```diff
@@ -76,10 +76,22 @@ lo (Graph ps w h c l)    _  = toText $ makeGraph
 lo (Cell _) _               = empty
 lo (CodeBlock _) _          = empty
 
+-- | Helper for converting layout objects into a more printable form.
+-- This function is specific to definitions.
+lo' :: LayoutObj -> PrintingInformation -> D
+lo' (HDiv _ con _)      sm = print' sm con
+lo' (EqnBlock contents) _  = makeEquation contents 0.8
+lo' obj sm                 = lo obj sm
+
 -- | Converts layout objects into a document form.
 print :: PrintingInformation -> [LayoutObj] -> D
 print sm = foldr (($+$) . (`lo` sm)) empty
 
+-- | Converts layout objects into a document form.
+-- Specific to Definitions.
+print' :: PrintingInformation -> [LayoutObj] -> D
+print' sm = foldr (($+$) . (`lo'` sm)) empty
```

Comment:
If this is `Specific to Definitions`, could it have a more meaningful name, such as `printDef`?

Reply:
Fixed in https://github.com/JacquesCarette/Drasil/pull/3910/commits/f78f58a2d6d93b74e7b82afe12e536ffedbacf8c.